orleans/JournaledTodoList/JournaledTodoList.WebApp/Grains/TodoListRegistryGrain.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Immutable;
+using Orleans.EventSourcing;
+using Orleans.Providers;
+using Orleans.Utilities;
+
+namespace JournaledTodoList.WebApp.Grains;
+
+/// <summary>
+/// A "state" based Journaled Grain. It does not save the events, just the state.
+/// </summary>
```

Comment:
I would like to add something here that tells Orleans to have one copy of this grain on each silo. Using `[StatelessWorker(1)]` didn't seem to work, or at least, it resulted in notificaitons/subscriptions not working as expected.

Reply:
Nevermind, I was looking at the [Replicated grains](https://learn.microsoft.com/en-us/dotnet/orleans/grains/event-sourcing/replicated-instances) features that are no longer available.